pages/linux/mknod.md

```diff
@@ -0,0 +1,19 @@
+# mknod
+
+> Create block or character device special files.
+
+- Create a block device:
+
+`sudo mknod {{path/to/device_file}} b {{major_device_number}} {{minor_device_number}}`
+
+- Create a character device:
+
+`sudo mknod {{path/to/device_file}} c {{major_device_number}} {{minor_device_number}}`
+
+- Create a FIFO (queue) device:
+
+`sudo mknod {{path/to/device_file}} p`
+
+- Create a device file with default SELinux security context:
+
+`sudo mknod -Z {{path/to/device_file}} {{type}} [{{major_device_number}} {{minor_device_number}}]`
```

Comment:
We don't use the `[` `]` syntax. Either mention the major and minor nos. or don't. Up to you whether you think it is important to keep.